scss/2-tools/_typography.scss

```diff
@@ -28,7 +28,8 @@
   .cads-prose img + &,
   .cads-prose details + &,
```

Comment:
Can we remove `.cads-prose details` at the same time (that's the _old_ targeted content)

Reply:
nice catch!